src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/__init__.py

```diff
@@ -8,7 +8,7 @@
 from azure.cli.core import AzCommandsLoader
 from azure.cli.core.commands import CliCommandType
 import azure.cli.command_modules.iot._help  # pylint: disable=unused-import
-from azure.cli.core.extension import extension_exists
+from azure.cli.core.extensions import extension_exists
```

Comment:
Even with extensions in core, if we are saying that these operations are usable and won't break, it needs to be included in and imported from `azure.cli.core.api`.